src/gdcdictionary/schemas/_terms.yaml

```diff
@@ -7499,3 +7503,79 @@ extrathyroid_extension:
       cde_id: 3179452
       cde_version: 1.0
       term_url: "https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=3179452%20and%20ver_nr=1"
+
+genome_doublings:
```

Comment:
There should be a naming discussion here. My thoughts are that that plurals indicate collections (e.g. Array types) and singular represents a single value. 
Should the be named genome_doubling_event_count ?

Reply:
@ZhenyuZ what are your thoughts?

Reply:
This is a good point, we do have have in our change request conventions that plural named properties should usually be reserved for array type properties: https://gdc-ctds.atlassian.net/wiki/spaces/GDC/pages/83525668/Change+Request+Conventions

Reply:
Per Zhenyu:  "genome_doubling" should be fine.

Reply:
@stilesj-uchicago any additional review before I merge this thing?

Reply:
You're good to go.